src/loaders/amf_load.c

```diff
@@ -264,12 +295,19 @@ static int amf_load(struct module_data *m, HIO_HANDLE *f, const int start)
 
 	/* Tracks */
 
-	trkmap = calloc(sizeof(int), mod->trk);
+	/* Index 0 is a blank track that isn't stored in the file. To keep
```

Comment:
I don't think it's really necessary to document the bugs of older XMP versions here (and further below).